components/style/binding_tools/check_bindings.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+
+def build(objdir, verbose=False):
+    with open(objdir + "/bindings.rs", "r") as bindings:
+        with open(objdir + "/check_bindings.rs", "w+") as tests:
+            tests.write("/* This Source Code Form is subject to the terms of the Mozilla Public\n")
+            tests.write(" * License, v. 2.0. If a copy of the MPL was not distributed with this\n")
+            tests.write(" * file, You can obtain one at http://mozilla.org/MPL/2.0/. */\n\n")
+            tests.write("/* automatically generated by check_bindings.py. */\n\n")
+            tests.write("fn assert_types() {\n")
+
+            pattern = re.compile("fn\s*Servo_([a-zA-Z0-9]+)\s*\(")
+
+            for line in bindings:
+                match = pattern.search(line)
+
+                if match:
+                    tests.write("    [ Servo_" + match.group(1) + ", bindings::Servo_" + match.group(1) + " ];\n")
+
+            tests.write("}\n")
```

Comment:
Instead, we could have a template for this above, like
``` python
TEMPLATE = '''
    [ Servo_{name}, bindings::Servo_{name} ];
'''
```

and `format` it every time we find a match.